sympy/geometry/ellipse.py

```diff
@@ -1005,6 +1005,45 @@ def semilatus_rectum(self):
         """
         return self.major * (1 - self.eccentricity ** 2)
 
+    def auxiliary_circle(self):
+        """Returns equation of auxiliary circle of the ellipse.
+
+        Auxiliary Circle of an ellipse is circumcircle of an ellipse i.e. the circle
+        whose center concurs with that of the ellipse and whose radius is equal to the ellipse's semi-major axis.
+
+        Parameters
+        ==========
+
+        x : str, optional
+            Label for the x-axis. Default value is 'x'.
+        y : str, optional
+            Label for the y-axis. Default value is 'y'.
+
+        Returns
+        =======
+
+        equation : sympy expression
+
+        Examples
+        ========
+
+        >>> from sympy import Circle, Ellipse, Point2D, Symbol
+        >>> e = Symbol('e')
+        >>> c = Symbol('c')
+        >>> e1 = Ellipse(Point2D(3, 8), 7, 8)
+        >>> e = e1.auxiliary_circle()
+        >>> e
+        Circle(Point2D(3, 8), 8)
+        >>> e.equation()
+        (x - 3)**2 + (y - 8)**2 - 64
+        >>> c1 = Circle(Point2D(2, 5), 4)
+        >>> c = c1.auxiliary_circle()
+        >>> c
+        Circle(Point2D(2, 5), 4)
+
+        """
+        return Circle(Point2D(self.center.x, self.center.y), Max(self.hradius, self.vradius))
```

Comment:
Can we just use self.centre instead of `Point2D(self.center.x, self.center.y)`?

Reply:
Yes, `self.center` works fine thanks @oscarbenjamin